packages/features/webhooks/lib/service/WebhookService.ts

```diff
@@ -0,0 +1,294 @@
+import { createHmac } from "crypto";
+
+import logger from "@calcom/lib/logger";
+import { WebhookTriggerEvents } from "@calcom/prisma/enums";
+
+import type { WebhookSubscriber, WebhookDeliveryResult } from "../dto/types";
+import type { WebhookPayload } from "../factory/types";
+import type { ITasker } from "../interface/infrastructure";
+import type { IWebhookRepository, IWebhookService } from "../interface/services";
+import { TaskerProvider } from "../provider/TaskerProvider";
+
+const log = logger.getSubLogger({ prefix: ["[WebhookService]"] });
+
+export class WebhookService implements IWebhookService {
+  constructor(
+    private readonly repository: IWebhookRepository,
+    private readonly tasker?: ITasker // Optional for fallback import
+  ) {}
+
+  protected async sendWebhook(
+    trigger: WebhookTriggerEvents,
+    payload: WebhookPayload,
+    subscriber: WebhookSubscriber
+  ): Promise<WebhookDeliveryResult> {
+    try {
+      if (process.env.TASKER_ENABLE_WEBHOOKS === "1") {
```

Comment:
Ideally we should inject this flag as well. Would be awesome when we would be able to unit test without mocking env variables too.